samples/client/petstore/scala-akka/build.sbt

```diff
@@ -14,10 +14,10 @@ libraryDependencies ++= Seq(
   "org.json4s" %% "json4s-jackson" % "3.6.7",
   "org.json4s" %% "json4s-ext" % "3.6.7",
   "de.heikoseeberger" %% "akka-http-json4s" % "1.27.0",
-  "org.scala-lang.modules" %% "scala-collection-compat" % "2.4.1",
+  "org.scala-lang.modules" %% "scala-collection-compat" % "2.12.0",
```

Comment:
Because of these warnings:
```
[WARNING] Scala library detected 2.12.13 doesn't match scala.compat.version : 2.4.1
[WARNING]  Expected all dependencies to require Scala version: 2.4.1
[WARNING]  org.openapitools:scala-akka-petstore-client:1.0.0 requires scala version: 2.12.13
[WARNING] Multiple versions of scala libraries detected!
```
Well, the warnings are still there, but 2.12.0 is closer to 2.12.13, right? ;-)
```
[WARNING] Scala library detected 2.12.13 doesn't match scala.compat.version : 2.12.0
[WARNING]  Expected all dependencies to require Scala version: 2.12.0
[WARNING]  org.openapitools:scala-akka-petstore-client:1.0.0 requires scala version: 2.12.13
[WARNING] Multiple versions of scala libraries detected!
```
Looks like the expectation (2nd line) is always the version of scala-collection-compat, but I can't find a matching version for all components, because...
- with 2.4.1 there's no Scala 2.4.x (anymore?)
- with 2.12.13 there's no scala-collection-compat 2.12.13 (the most recent one is 2.12.0)
- with 2.12.0 there's a new warning popping up:
```
[WARNING]  com.typesafe.akka:akka-actor_2.12:2.6.12 requires scala version: 2.12.11
```
So I thought with Scala 2.12.13 and scala-collection-compat 2.12.0 the versions differ the least.

Reply:
I'm just wondering why changes to the Scala libraries are necessary as you were only fixing JUnit here which is a Java library? So I would try to avoid touching anything else?